### PR TITLE
Add Tomoscan review page.

### PR DESCRIPTION
**Pull request by Leoloup**
This is a rough draft version.

### CODE REVIEW

**Comment by etienne-napoleone** [comment]
Waiting for update after talking about it on telegram

**Comment by Leoloup** [comment]
> 
> 
> Waiting for update after talking about it on telegram

2nd version is out. I struggle a little bit with the pictures, It seems that I don't know how to link properly the pictures...

File: docs/tomoscan/Tomoscan.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+![alternative textuelle](/whole block tomoscan.jpg)
+
+The informations available are the following :
+-   Height : the number of the block.
+-   Age : the time where the block was mined.
+-   Txn : number of transactions made on this block.
+-   Miner : the adress of the miner that verified the block.
+-   GasUsed: the nergy needed to mine the block.
+-   Finality : it is the guarantee that past transactions can never change. If it is higher, the security is better.
```
Comment (etienne-napoleone) [inline docs/tomoscan/Tomoscan.md]:
```suggestion
- Finality: The percentage of the network who validated this block. When it reach 75%, the block reach it's finality state and is added permanently to the chain.
```

File: docs/tomoscan/Tomoscan.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+If you click on a block, you will find more informations such as more detailled transactions made on the block and the Blocksigner. Blocksigner is the Masternode that created the block.
```
Comment (etienne-napoleone) [inline docs/tomoscan/Tomoscan.md]: You can make that part more detailed

File: docs/tomoscan/Tomoscan.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+![alternative textuelle](/assets/block detailled tomoscan.jpg)
+
+### Sign and Register
+
+If you register, you will be able to keep a watchlist of some adressesses, keep them on a list and receive a notification when by email when a transaction is made with this adress,
```
Comment (etienne-napoleone) [inline docs/tomoscan/Tomoscan.md]:
```suggestion
If you register, you will be able to keep a watchlist of some addresses your interested into.
You will receive an email notification when this address perform actions.
```

File: docs/tomoscan/Tomoscan.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Please keep in mind that Tomoscan is subject to go through some changes. That is why your feedback is appreciated!
```
Comment (etienne-napoleone) [inline docs/tomoscan/Tomoscan.md]:
```suggestion
Please keep in mind that TomoScan is subject to go through some changes. That is why your feedback is appreciated!
```
We always capitalise the second word like: Tomo**C**hain, Tomo**S**can, Tomo**W**allet

File: docs/tomoscan/Tomoscan.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+You can check all the blocks that were mined on Tomochain using the Search button. 
+
+![alternative textuelle](/whole block tomoscan.jpg)
+
+The informations available are the following :
+-   Height : the number of the block.
+-   Age : the time where the block was mined.
+-   Txn : number of transactions made on this block.
+-   Miner : the adress of the miner that verified the block.
```
[the remark this answers is not in the document]
Comment (Leoloup) [reply docs/tomoscan/Tomoscan.md]: wow, my bad for that one.

**Comment by Leoloup** [comment]
Add 3rd version. It still needs to be polished though.

File: docs/tomoscan/TomoScan.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+TomoScan provides a block explorer with user friendly, detailed and professional user interface to the TomoChain ecosystem. 
+TomoScan brings TomoChain’s transparency to users. 
+All blocks, transactions, smart contracts, tokens and basically everything happening on the chain are shown to the users.
+Furthermore, TomoScan also offers technical overview and expose useful metrics about the TomoChain. 
+
+Here are the pages and features available on TomoScan:
+
+## [Home](https://scan.testnet.tomochain.com/)
+      
+![TomoScan homepage](/assets/tomoscan_home.jpg)
```
Comment (etienne-napoleone) [inline docs/tomoscan/TomoScan.md]: This is a duplicate

File: docs/tomoscan/TomoScan.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+- The sender's address.
+- The recipient's address.
+- The transaction's ammount in TOMO.
+- The gas used.
+- The gas price.
+- The total gas cost.
+- The transaction data.
+
+## [Accounts](https://scan.testnet.tomochain.com/accounts)
+List the accounts on the TomoChain.[TomoScan accounts](/assets/TomoScan_account.jpg)
```
Comment (etienne-napoleone) [inline docs/tomoscan/TomoScan.md]:
```suggestion
List the accounts on TomoChain.
[TomoScan accounts](/assets/TomoScan_account.jpg)
```
Also, if you add a picture here, please add it to each top level categories too for consistency.

File: docs/tomoscan/TomoScan.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+ The Accounts part is divided in three parts : all accounts, all masternodes and verified contracts. Various informations can be found:
+ 
+### [All accounts](https://scan.testnet.tomochain.com/accounts)
+- The rank: it is based on the number of TOMO the account owns. The more TOMO the account has on his balance, the higher he will be.
+- The adress of the account.
+- The balance of TOMO.
+- The number of transactions made on this account.
+
+### [All masternodes](https://scan.testnet.tomochain.com/masternodes)
+- The adress of a masternode.
```
Comment (etienne-napoleone) [inline docs/tomoscan/TomoScan.md]: In general, prefer this form:
```suggestion
- The masternode's address.
```

File: docs/tomoscan/TomoScan.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+
+The informations available for each block are the following :
+- Height: the block number.
+- Age: How long ago the block was created.
+- Txn: number of transactions in this block.
+- Miner: the address of the masternode who created this block.
+- GasUsed: the gas used to mine the block.
+- Finality: The percentage of the network who validated this block. When it reach 75%, the block reach it's finality state and is added permanently to the chain.
+
+If you click on a block, you will find more detailled informations such as :
```
Comment (etienne-napoleone) [inline docs/tomoscan/TomoScan.md]: This should be on a separate `##` level called "Block" (see "Transaction")

**Pull request merged**